src/csharp/Grpc.Core/GrpcOperationFailedException.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+namespace Grpc.Core
+{
+    public class GrpcOperationFailedException : InvalidOperationException
```

Comment:
FTR, we actually uses to have a similar exception type in a way earlier version of gRPC C# (pre-alpha), but then got rid of it.